index.html

```diff
@@ -265,6 +265,42 @@ <h3>CSS</h3>
 
   </div>
 
-  
+  <section class="contact" id="Contact">
```

Comment:
- [ ] Instead of `<section>` element here, Please use `<footer>` tag. It will make sure your code is semantic and follows HTML best practices.